virttest/utils_test/libvirt.py

```diff
@@ -1892,7 +1892,13 @@ def file_exists():
 
     # Wait for file write over with '</disk>' keyword at the file end
     wait_for_file_over("</disk>", diskxml.xml)
-    return diskxml.xml
+
+    fd, path = tempfile.mkstemp(prefix="tmp_xml", suffix=".xml")
```

Comment:
The function tempfile.mkstemp()  is insecurity, so do not recommend to use it. 
You might use data_dir.get_tmp_dir(), then you need not to delete the temp file after the test, either.

Reply:
Done!